test/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "test": "node --preserve-symlinks test.js test --framework karma",
+    "debug": "node --preserve-symlinks test.js test --framework karma --log-level debug",
```

Comment:
I question if this sort of thing should be added.  It's certainly useful when debugging, but we don't want to add this to every package.  Once the current debugging is over, this might never be used again.

One solution is to just add that by hand when you need it:
`npm run test -- --log-level debug`

Another could be to have a general log level override via an env var?  So for any package you could do:
`BEDROCK_LOG_LEVEL=debug npm t`